### ISSUE TITLE
64-bit map keys

### ISSUE THREAD

**Issue by glycerine** [gijit/gi#12]
in order to enable 64-bit ints to be map keys, we may need to go to using an lds[1] like map.  The key 3LL, aka int64(3), doesn't seem to lookup in regular Lua tables. Update: confirmed; explanation here http://luajit.org/ext_ffi_semantics.html#cdata_key

~~~
LuaJIT 2.1.0-beta3 -- Copyright (C) 2005-2017 Mike Pall. http://luajit.org/                     
JIT: ON SSE2 SSE3 SSE4.1 BMI2 fold cse dce fwd dse narrow loop abc sink fuse                    
> a=3LL                                                                                         
> b={[3LL]=5LL}                                                                                 
> print(b[a])                                                                                   
nil                                                                                             
>  
~~~

[1] https://github.com/neomantra/lds  
>lds provides data structures which hold LuaJIT cdata.
>
>These containers cover the common use cases of Lua tables:
>
>Array (fixed-size array)
>Vector (dynamically-sized array)
>HashMap (key-value store)

**Comment by glycerine** [comment]
Or just wrap Go maps, slices, and arrays using reflection.

**Comment by glycerine** [comment]
fixed in v0.7.1 by translating map keys to Lua strings.
